events/events.go

```diff
@@ -170,8 +170,9 @@ func (ValidTx) GetChannel() ChannelID {
 
 // RewardReceived signals reward has been received
 type RewardReceived struct {
-	Coinbase string
-	Amount   uint64
+	Coinbase  string
```

Comment:
The only place this RewardReceived struct is used is for the Pubsub. It did not contain a layer before, so I didn't change it. Since this is not really used in the DB, is the layer needed at all?

Reply:
Okay. No, it's fine, we're not using pubsub in production for anything right now. It's confusing that we have multiple structs but let's leave this alone for now.